src/Frontend/Core.hs

```diff
@@ -307,11 +308,11 @@ instance Show a => ToHtml (PageShow a) where
     toHtmlRaw = toHtml
     toHtml = pre_ . code_ . toHtml . ppShow . _unPageShow
 
-data CommentVotesWidget = VotesWidget CommentContext Comment
+data CommentVotesWidget = VotesWidget [IdeaCapability] CommentContext Comment
```

Comment:
So far we've created the idea caps lists locally in the ToHtml instances, now we are beginning to pass them around.  Perhaps we should consistently keep them in the RenderContext?  It doesn't alway apply there, though.  Should RenderContext become polymorphic in the page type then?

Probably best to ponder this some more before we change anything.